content/docs/deploy/core/upgrading.mdx

```diff
@@ -29,7 +35,13 @@ sidebar_position: 10
 
 - Internal [RDS changes](https://github.com/pomerium/pomerium/pull/4098) reduce memory consumption, especially for environments where configuration changes rapidly.
 
-## Since 0.20.0
+## 0.20.1
```

Comment:
Would it make more sense to relabel these headings in a separate PR, as this isn't specifically related to v0.22.3?

Reply:
We could, I just relabeled those based on Bobby's comment in Slack about updating headers. I can scrap this PR and redo it.